ts_src/confidential.ts

```diff
@@ -1,8 +1,16 @@
+import { WitnessUtxo } from 'bip174/src/lib/interfaces';
 import * as secp256k1 from 'secp256k1-zkp';
 
 import * as bufferutils from './bufferutils';
 import * as crypto from './crypto';
 
+export interface BlindingData {
+  satoshis: number;
+  amountBlinder: string;
+  asset: string;
+  assetBlinder: string;
+}
```

Comment:
Do we really need this? Can't we use the [UnblindOutputResult](https://github.com/vulpemventures/liquidjs-lib/blob/master/ts_src/confidential.ts#L45) type? This is what we do in go-elements.. want to keep these 2 confidential modules as much similar as possible.

Reply:
I agree but IMO we should keep BlindingData (use strings) instead of UnblindOutputResult (use Buffers) in order to have a "green-like" interface: https://github.com/vulpemventures/liquidjs-lib/issues/9#issue-800167296

Reply:
well, I don't understand why for go-elements we don't have these concerns while here we want to complicate things.. let's just use UnblindOutputResult, the green-like interface can be easily transformed into that type after all.